src/Bindings/ManualBindings.cpp

```diff
@@ -3727,6 +3780,11 @@ void cManualBindings::Bind(lua_State * tolua_S)
 			tolua_function(tolua_S, "SendPluginMessage", tolua_cClientHandle_SendPluginMessage);
 		tolua_endmodule(tolua_S);
 
+		tolua_beginmodule(tolua_S, "cColor");
+			tolua_function(tolua_S, "Set", tolua_cColor_Set);
+			tolua_function(tolua_S, "Get", tolua_cColor_Get);
```

Comment:
I didn't do per-component getter/setter functions because it'd be a lot of duplicated code and I couldn't think of a use case (maybe a plugin that had R/G/B sliders? But even if someone found a usecase, they could write a short workaround function).

```
function SetRed(Item, R)
   local _, G, B = Item.m_ItemColor:Get()
   Item.m_ItemColor:Set(R, G, B)
end
```